### PR TITLE
[StateCodelab] Bump to alpha08

### PR DESCRIPTION
**Pull request by objcode**

### CODE REVIEW

File: StateCodelab/build.gradle [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
     }
-    ext.kotlin_version = "1.4.10"
+    ext.kotlin_version = "1.4.20"
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha16"
+        classpath "com.android.tools.build:gradle:4.2.0-beta01"
```
Comment (yrezgui) [inline StateCodelab/build.gradle:28]: Use `com.android.tools.build:gradle:7.0.0-alpha02` instead

**Review by yrezgui — CHANGES_REQUESTED** [review]
You need to update AGP (check comment) and Gradle to 6.7.1

File: StateCodelab/build.gradle (hunk 1 above)
Comment (objcode) [reply StateCodelab/build.gradle]: Done! PTAL

**Review by yrezgui — APPROVED** [review]

**Comment by yrezgui** [comment]
I'm merging the PR. I'll update the GitHub action as it needs to set the environment to Java 11

**Pull request merged**